docs/bsp.md

```diff
@@ -725,7 +735,7 @@ trait CompileReport {
   def warnings: Int
 
   /** The total number of milliseconds it took to compile the target. */
-  def time: Option[Int]
+  def time: Option[Long]
```

Comment:
Long does fit in a JSON number. Let's turn this into a Double instead.

Reply:
I'm not sure we should make this a double since it will store most likely ms. What I want here is a type representing the biggest integer possible.
  
The question is: should we type things as in Scala or as JSON allows? Another way to phrase it: is `Int` everywhere in the proposal representing an integer following the JSON spec?
  
If that's the case, we should probably add a paragraph explaining that.